apps/src/code-studio/pd/foorm/editor/FoormSaveBar.jsx

```diff
@@ -81,6 +81,21 @@ const publishedSaveWarning = (
   </div>
 );
 
+const aboutToPublishWarning = (
+  <div>
+    <span style={styles.warning}>Warning: </span>You are about to publish a new
+    survey. Once a survey is published, it may be put into active use.
```

Comment:
nit: do we want to add to the warning that you cannot go back to draft after you have published a survey?